### PR TITLE
zh: add more config description

### LINKED ISSUE #1139: Some common fields like affinity, toleration, etc. are missed in the configuration doc
## Error Report

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?

https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster
Some common fields like affinity, toleration, etc. are missed in the configuration doc, we have to check all the missed fields and evaluate how to cover them in the doc.
After the above change, we have to update the deployment docs of TiFlash, TiCDC, Pump, etc. to refer to the configuration doc to customize some fields if necessary.

2. How would you like to improve it?

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Close #1139 
Close https://github.com/pingcap/docs/issues/5069

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -515,6 +522,61 @@ TiDB 是分布式数据库，它的高可用需要做到在任一个物理拓扑
...
+    # ...
+  tidb:
+    nodeSelector:
+      node-role.kubernetes.io/tidb: true
+    # ...
+```
+
+#### 通过 tolerations 调度实例
+
+通过各组件配置的 `tolerations` 字段，可以允许组件的实例能够调度到带有与之匹配的污点的节点上。关于污点与容忍度的更多说明，请参阅 [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)。
```
Comment (ran-huang) [inline zh/configure-a-tidb-cluster.md:550]:
```suggestion
通过各组件配置的 `tolerations` 字段，可以允许组件的实例能够调度到带有与之匹配的[污点](https://kubernetes.io/docs/reference/glossary/?all=true#term-taint) (Taint) 的节点上。关于污点与容忍度的更多说明，请参阅 [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**